preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -241,11 +238,18 @@ public PreferencesFx addEventHandler(EventType<PreferencesFxEvent> eventType, Ev
    *
    * @param eventType    the event type from which to unregister
    * @param eventHandler the handler to unregister
-   *
    * @throws NullPointerException if either event type or handler are {@code null}.
    */
   public PreferencesFx removeEventHandler(EventType<PreferencesFxEvent> eventType, EventHandler<? super PreferencesFxEvent> eventHandler) {
     preferencesFxModel.removeEventHandler(eventType, eventHandler);
     return this;
   }
+
+  public PreferencesFxView getPreferencesFxView() {
```

Comment:
I think `getView()` should be enough as a name, since it's being called on a `PreferencesFx` object already

Reply:
👍